### DISCUSSION TITLE
No permite autenticar, dice que no se está registrado

### DISCUSSION THREAD

**Discussion by santiago-collab** [rramosp/ai4eng.v1#246] [00 - General]
Buenas tardes, profe, cómo está?

Lo que pasa es que intento verificar mi usuario para envío de actividad, pero me dice que no estoy registrado, verifico en plataforma y si tengo mi usuario con las notas, entonces quisiera preguntar, por favor, qué puedo hacer para autenticar mi usuario
<img width="953" alt="Sin título2" src="https://user-images.githubusercontent.com/68567878/223839555-e4ad1c55-f16b-4988-b0f3-26f4e0ee51c6.png">
<img width="673" alt="Sin título" src="https://user-images.githubusercontent.com/68567878/223839559-47ac8ec2-ec8c-48ad-b904-b1a63cdde971.png">

**Comment by imEag** [comment]
Hola Santiago.
1. ¿Es la primera vez que te ocurre este error? 
2. ¿Cuándo fue la ultima vez que enviaste una respuesta con este usuario?
3.  Regálame el correo que estas usando.

**Reply by Felipe-RA** [reply]
Adicional a lo que comenta @imEag 

 @santiago-collab Veo que estas registrado con un correo no institucional y en la **versión abierta del curso para personas fuera de la UdeA**. ¿Eres un estudiante de la UdeA?

Si es el caso dinos y te damos instrucciones extra.

Hasta pronto

**Reply by santiago-collab** [reply]
> Hola Santiago.
> 
> 1. ¿Es la primera vez que te ocurre este error?
> 2. ¿Cuándo fue la ultima vez que enviaste una respuesta con este usuario?
> 3. Regálame el correo que estas usando.

Hola, muy buenos días. 

1. Si señor, primera vez que me sucede
2. La última vez que envié respuesta con este usuario fue hace un año y medio
3. santiago_200012@hotmail.com

**Reply by santiago-collab** [reply]
> Adicional a lo que comenta @imEag
> 
> @santiago-collab Veo que estas registrado con un correo no institucional y en la **versión abierta del curso para personas fuera de la UdeA**. ¿Eres un estudiante de la UdeA?
> 
> Si es el caso dinos y te damos instrucciones extra.
> 
> Hasta pronto

Si señor, estoy registrado con la versión abierta, no soy estudiante de la UdeaA, sino que tomé el curso en su modalidad abierta. Muchas gracias por su apoyo.

**Reply by gitogel** [reply]
Hola, me sucede lo mismo, ¿pudiste solucionarlo?

**Reply by Felipe-RA** [reply]
@gito@santiago-collab 

Estamos mirando cuál puede ser el problema.

**Reply by gitogel** [reply]
Hola, ¿alguna respuesta?

**Comment by rramosp** [comment]
estamos viendo que en general està funcinando bien. no podemos ver el
error. puedes probar por favor desde otra direccón de correo y nos cuentas?

On Mon, Mar 13, 2023 at 7:43 AM gitogel ***@***.***> wrote:

> Hola, ¿alguna respuesta?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/246#discussioncomment-5291064>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIK2GFLTIAZAQUQXDLLW34I53ANCNFSM6AAAAAAVUGXRWQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Comment by gitogel** [comment]
Hola:
Con otra cuenta funcionó bien.
Saludos cordiales,
Gelza
